diagnostics-extension/package.json

```diff
@@ -29,6 +30,7 @@
     "@types/jest": "^26.0.23",
     "@typescript-eslint/eslint-plugin": "^4.28.1",
     "@typescript-eslint/parser": "^4.28.1",
+    "cros-dpsl-js": "file:../../telemetry-support-extension-for-chromeos/cros-dpsl-js-1.0.1.tgz",
```

Comment:
This should be 1.0.1 once the new version is published.

This is causing the check to fail.